core/internal/mocks/client.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.7.5. DO NOT EDIT.
+// Code generated by mockery v0.0.0-dev. DO NOT EDIT.
```

Comment:
was this generated by some custom version of mockery? maybe better revert these files